src/main/resources/META-INF/resources/webjars/service/prov/prov.js

```diff
@@ -2066,7 +2066,7 @@ define(function () {
 					success: updatedCost => current.defaultCallback(type, updatedCost)
 				});
 			});
-			$('.prov-project .icon').attr('class',`fa-fw ${current.model.node.tool.uiClasses}`);
+			$('.service-icon').attr('class',`fa-fw ${current.model.node.tool.uiClasses}`);
```

Comment:
You still override the class attribute of the element. The selector should not be based on the attribute you change.